services/brig/src/Brig/Schema/V_FUTUREWORK.hs

```diff
@@ -17,7 +17,7 @@
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
 
-module V_FUTUREWORK
+module Brig.Schema.V_FUTUREWORK
```

Comment:
This module sounds like it needs some love. Can you please create a ticket to address it?